waspc/tests/Generator/WebAppGeneratorTest.hs

```diff
@@ -98,7 +98,10 @@ spec_WebAppGenerator = do
     --   that they will successfully be written, it checks only that their
     --   destinations are correct.
     it "Given a simple AppSpec, creates file drafts at expected destinations" $ do
-      let (_, Right fileDrafts) = runGenerator $ genWebApp testAppSpec
+      let (_, generatorResult) = runGenerator $ genWebApp testAppSpec
+      fileDrafts <- case generatorResult of
+        Right fds -> pure fds
+        Left err -> expectationFailure ("genWebApp failed: " ++ show err) >> error "unreachable"
```

Comment:
```suggestion
      fileDrafts <- case (snd $ runGenerator $ genWebApp testAppSpec) of
        Right fds -> pure fds
        Left err -> expectationFailure ("genWebApp failed: " ++ show err) >> error "unreachable"
```

but yeah would maybe be even nicer if you used `fromRight` or something (not sure if doable here). This is also not terrible though.

Reply:
Used `fromRight`.